motionlayout/src/main/java/com/google/androidstudio/motionlayoutexample/viewpagerdemo/ViewPagerAdapter.kt

```diff
@@ -17,11 +17,11 @@
 package com.google.androidstudio.motionlayoutexample.viewpagerdemo
 
 import android.os.Bundle
-import android.support.v4.app.Fragment
-import android.support.v4.app.FragmentManager
-import android.support.v4.app.FragmentPagerAdapter
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
 
-class ViewPagerAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm) {
+class ViewPagerAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm!!) {
```

Comment:
Please don't use !! if possible. I would change the type of fm to FragmentManager

Reply:
It is mended per your suggestion. Please review latest commit